usaspending_api/etl/management/commands/load_subawards.py

```diff
@@ -60,17 +62,29 @@ def handle(self, *args, **options):
         else:
             db_cursor = PhonyCursor()
 
-        submission_ids_to_update = []
+        submissions_to_update = []
 
         if options["update_all"]:
             submissions_to_update = SubmissionAttributes.objects.exclude(broker_submission_id__isnull=True)
         else:
-            submission = SubmissionAttributes.objects.filter(broker_submission_id=options['submission_id']).first()
-            if not submission:
-                logger.critical("Submission {} not found in datastore".format(options['submission_id']))
-                return
-            submissions_to_update = [submission]
+            for submission_id in options['submission_id']:
+                sub = SubmissionAttributes.objects.filter(broker_submission_id=submission_id).first()
+                if not sub:
+                    logger.critical("Submissions not found in datastore".format(options['submission_id']))
+                else:
+                    submissions_to_update.append(sub)
 
+        failed_submissions = []
+        success_submissions = []
         for submission in submissions_to_update:
-            logger.info("Loading subaward data for submission {}".format(submission.broker_submission_id))
-            load_subawards(submission, db_cursor)
+            try:
```

Comment:
Not particularly, any exception in the loading will be needed to investigate manually, so catching all exceptions, logging it, and moving to the next submission is probably the best course of action when loading a large set of submissions' subawards.